content/docs/command-reference/commit.md

```diff
@@ -132,9 +132,9 @@ $ dvc pull -aT
 Sometimes we want to iterate through multiple changes to configuration, code, or
 data, trying different ways to improve the output of a stage. To avoid filling
 the <abbr>cache</abbr> with undesired intermediate results, we can run a single
-stage with `dvc run --no-commit`, or reproduce an entire pipeline using
-`dvc repro --no-commit`. This prevents data from being pushed to cache. When
-development of the stage is finished, `dvc commit` can be used to store data
+stage with `dvc repro --no-commit <target>`, or reproduce an entire pipeline
+with `dvc repro --no-commit`. This prevents data from being pushed to the cache.
```

Comment:
I'd combine these into a single one (probably no need to differentiate between single or multiple stages) but OK, not the direct purpose of this PR.

Reply:
I've merged them, could you PTAL?

Reply:
Thanks, I'll leave a suggestion that further generalizes this. ⌛